src/rpc/rawtransaction.cpp

```diff
@@ -1649,6 +1663,16 @@ static RPCHelpMan decodepsbt()
             out.pushKV("blinder_index", (int64_t)*output.m_blinder_index);
         }
 
+        // Blind value proof
+        if (!output.m_blind_value_proof.empty()) {
+            out.pushKV("blind_value_proof", HexStr(output.m_blind_value_proof));
+        }
+
+        // Blind asset proof
+        if (!output.m_blind_value_proof.empty()) {
```

Comment:
copy/paste error: `m_blind_value_proof` should be `m_blind_asset_proof`

Reply:
Fixed